app/backend/src/couchers/models/User.py

```diff
@@ -0,0 +1,219 @@
+import enum
```

Comment:
Please use lowercase filenames in `snake_case`

Reply:
fixed